eg/servo-slider.js

```diff
@@ -1,14 +1,12 @@
-var five = require("../lib/johnny-five.js");
-var board = new five.Board();
+const {Board, Sensor, Servo} = require("../lib/johnny-five.js");
+const board = new Board();
 
-board.on("ready", function() {
+board.on("ready", () => {
 
-  var slider = new five.Sensor("A0");
-  var tilt = new five.Servo(9);
+  const slider = new Sensor("A0");
+  const tilt = new Servo(9);
 
-  slider.scale([0, 180]).on("slide", function() {
-
-    // The slider's value will be scaled to match the tilt servo range
-    tilt.to(this.value);
+  slider.on("slide", () => {
+    tilt.to((slider.scaleTo(0, 180)));
```

Comment:
There's a set of unnecessary parens in there. Have you been writing C?

Reply:
I'm sure it was my editor "helping"